### ISSUE TITLE
Accessibility issues with our theme builder

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#3226]
This is a collection of all accessibility issues that i could find when testing. 
## Startpage
- [x] Switch example do not have a label
![Image](https://github.com/user-attachments/assets/5630dcdf-f9a8-4f73-a952-7f7d09b02445)
- [x] Blir lest "a11y title" på knapper. Ser ut til at det kommer fra en `aria-labeledby` på iconet i button. Tror ikke vi trenger en `aria-labeledby` på iconet. Den kan eventuelt stå tom om det er mulig. Du får all informasjon fra lenketeksten.
````
<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" fill="none" viewBox="0 0 24 24" focusable="false" role="img" aria-labelledby="title-r19" font-size="1.5rem">
<title id="title-r19">a11y-title</title>
````

### Enhancements
- [x] Legg til aria-label for hvem du følger i eksempelet. Nå får du tre knapper som sier følg uten kontekst. 

## Oversikt/Fargesystem
- [x] Tab order. When using a keyboard you have to tab trough the whole example page to get to the panel for editing colors/border radius
- [x] There are no `main` landmark. The jump to main content button do not work
- [x] Change color buttons. Can we do anything with them? The have the same "a11y title" label as the startpage. Maybe give them a better name?
- [x] "Legg til" Add a legg til hovedfarge for the accessible name? 
- [x] "Gå tilbake" button inside a color has the same "a11y title"
- [x] Select have no label (Contrast color selects) 
  - The first select is fixed. The second where you choose for base colors do also need a label
![Image](https://github.com/user-attachments/assets/d0db9ab9-d36f-417b-8ca3-f59419c9e28a)
- [x] Border-radius button group should maybe have `aria-labelby`
- [x] Tab-order when entering a color in the setting panel is wrong. You have to tab trough elements in the last view to get to settings for the color you selected
- [x] Use `h1` headings 

 
- [x] All color buttons do not have a label

![Image](https://github.com/user-attachments/assets/f0c7cd5f-1199-48f9-b1bb-3d58646be0dd)

## Other
- [x] We could add some empty alt text to icon in the global light/dark mode button. Sortsite reads this as an error.
